pkg/cmd/server/kubernetes/master_config.go

```diff
@@ -43,7 +43,7 @@ import (
 )
 
 // AdmissionPlugins is the full list of admission control plugins to enable in the order they must run
-var AdmissionPlugins = []string{"NamespaceLifecycle", "OriginPodNodeEnvironment", "LimitRanger", "ServiceAccount", "SecurityContextConstraint", "BuildDefaults", "BuildOverrides", "ResourceQuota", "SCCExecRestrictions"}
+var AdmissionPlugins = []string{"NamespaceLifecycle", "OriginPodNodeEnvironment", "PodNodeConstraints", "LimitRanger", "ServiceAccount", "SecurityContextConstraint", "BuildDefaults", "BuildOverrides", "ResourceQuota", "SCCExecRestrictions"}
```

Comment:
OriginPodNodeEnvironment and PodNodeConstraints will fight with each other... OriginPodNodeEnvironment will force a pod to have a nodeSelector set which matches the project's nodeSelector, and PodNodeConstraints will forbid the pod from having a node selector. This is another case of needing a magic ordering between admission plugins :-/